tests/client/json_api_client_test.py

```diff
@@ -12,32 +12,29 @@
 
 class JsonApiClientTest(unittest.TestCase):
     def test_pack_params(self):
-        self.assertDictEqual(JsonApiClient._pack_params({
+        self.assertItemsEqual(JsonApiClient._pack_params({
```

Comment:
`assertItemsEqual(a, b)` checks whether `sorted(a) == sorted(b)`, so the ordering is already discarded.
On Aug 1, 2016 16:16, Stephen Koo <notifications@github.com> wrote:In tests/client/json_api_client_test.py:
> @@ -12,32 +12,29 @@
>  
>  class JsonApiClientTest(unittest.TestCase):
>      def test_pack_params(self):
> -        self.assertDictEqual(JsonApiClient._pack_params({
> +        self.assertItemsEqual(JsonApiClient._pack_params({

Perhaps wrap both arguments to this assertion with set() since the order is just an implementation detail?

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.